### PR TITLE
Renew certs

### PR DESCRIPTION
**Pull request by mikevanhemert**

### CODE REVIEW

File: .github/workflows/package.yaml [hunk 1]
```diff
@@ -87,7 +86,6 @@ jobs:
             --confirm \
             --tmpdir $PWD/minimal-tmp \
             --skip-sbom \
-            --no-local-images \
```
Comment (Vandiver247) [inline .github/workflows/package.yaml:90]: Why remove the `--no-local-images` line?

File: .github/workflows/package.yaml [hunk 2]
```diff
@@ -48,8 +48,7 @@ jobs:
             --set BRANCH="${{ env.BRANCH_NAME }}" \
             --confirm \
             --tmpdir $PWD/big-tmp \
-            --skip-sbom \
-            --no-local-images
```
Comment (Vandiver247) [inline .github/workflows/package.yaml:52]: Why remove the `--no-local-images` line?

File: .github/workflows/package.yaml (hunk 1 above)
Comment (Vandiver247) [reply .github/workflows/package.yaml:90]: NVM. Did not realize that flag is removed in this version.

File: .github/workflows/package.yaml (hunk 2 above)
Comment (Vandiver247) [reply .github/workflows/package.yaml:52]: NVM. Did not realize that flag is removed in this version.

**Review by Vandiver247 — APPROVED** [review]

File: .github/workflows/package.yaml (hunk 1 above)
Comment (mikevanhemert) [reply .github/workflows/package.yaml:90]: This [flag was removed](https://github.com/defenseunicorns/zarf/commit/b3d806703c35ded74dbced04bb4e1aae9ffa401a#diff-0fe6b3bdc0c719d3cf7c590e4c9bffc4f0d42afd394ed891dcb4e8f0697e11f5L259) in February. I'm not sure which version this commit got into but the pipeline definitely failed when we tried running with `v0.25.0-rc1`

File: .github/workflows/package.yaml (hunk 2 above)
Comment (mikevanhemert) [reply .github/workflows/package.yaml:52]: This [flag was removed](https://github.com/defenseunicorns/zarf/commit/b3d806703c35ded74dbced04bb4e1aae9ffa401a#diff-0fe6b3bdc0c719d3cf7c590e4c9bffc4f0d42afd394ed891dcb4e8f0697e11f5L259) in February. I'm not sure which version this commit got into but the pipeline definitely failed when we tried running with `v0.25.0-rc1`

**Pull request merged**